dwst/scripts/controller/error.js

```diff
@@ -14,6 +14,32 @@
 import errors from '../lib/errors.js';
 const {NoConnection, AlreadyConnected, SocketError, InvalidParticles, InvalidArgument, InvalidCombination, UnknownCommand, UnknownInstruction, UnknownHelpPage, UnknownText, UnknownBinary} = errors;
 
+function commaCommaOr(stringList) {
+  if (stringList.length === 0) {
+    return '';
+  }
+  if (stringList.length === 1) {
+    return stringList[0];
+  }
+  const last = stringList[stringList.length - 1];
+  const listExcludingLast = stringList.slice(0, stringList.length - 1);
+  const upToSecondLastJoined = listExcludingLast.join(', ');
+  return `${upToSecondLastJoined} or ${last}`;
+}
+
+// WAITING relocation descision ^
+
+function maybeCommaCommaOr(textOrList) {
+  if (typeof textOrList === 'object') {
+    return commaCommaOr(textOrList);
+  }
+  if (typeof textOrList === 'string') {
```

Comment:
This is dead code. The argument is never a string.

Reply:
Even if it was as string sometimes, I would just tell the user to wrap it in square brackets.